### ISSUE TITLE
Indent rule does not allow variable declaration with aligned names

### ISSUE THREAD

**Issue by targos** [eslint/eslint#3164]
Since v1.0.0-rc-1, the indent rule has a new behavior regarding variable name alignment in multiline declaration.
With `indent: [2, 2]` it is currently allowed to write code like the following:
```js
var original = p,
    seenLinks = {},
    knownHard = {};
```
However in v1.0.0, this results in an error:
```
  1387:5   error  Expected indentation of 2 characters  indent
  1388:5   error  Expected indentation of 2 characters  indent
```

Is that an intended change?
If so, I think a rule setting to allow that would be very useful.

**Comment by nzakas** [comment]
Yes, that is intended. You specified you want an indent of 2 spaces, but you used 4.

**Comment by gyandeeps** [comment]
As its working as expected so closing.
